### DISCUSSION TITLE
inappropriate bumping of libglm

### DISCUSSION THREAD

**Discussion by marekr** [microsoft/vcpkg#34511] [General]
We just stumbled across an issue here due to https://github.com/microsoft/vcpkg/commit/a880596395565021997e93daa176a4aa3b224f49

which kept the 0.9.9.8 "version" for the port but moved the commit pointer way ahead in the glm tree. This commit now introduces API breaking changes. 
In particular, glm has changed the data struct of quaternions from xyzw to wxyz order.

This is kinda inappropriate to do this to a port. Yes, we are forced to pin the package now, but this throws people off when you expect version 0.9.9.8 of a package to be 0.9.9.8 of a package. The port version variants should just be build related fixes.

**Comment by Osyotr** [comment]
> which kept the 0.9.9.8 "version" for the port but moved the commit pointer way ahead in the glm tree

Huh? https://github.com/microsoft/vcpkg/pull/32685/files#diff-ec6a64e2a9ccc66428e084a7cd370f33c4a8ce28f0be58ba8b2257e74ecfa3e2R3

Personally I think that vcpkg should use latest release unless other package managers agree on some other version.
https://repology.org/project/glm/versions

**Reply by marekr** [reply]
Derp, the things got filtered up to me internally a little wrong.

But yea, the point still relates there. It's not an official release and glm is clearly maintained, they just haven't released a new version.

Without them releasing a new version officially, documentation is lacking on the breaking changes
